src/cli.js

```diff
@@ -217,13 +215,22 @@ async function signInToThirdPartyTools(configs) {
 /**
  * The main execution function for tyr.
  */
-export default async function run() {
+export default async function run(tyr) {
```

Comment:
If `tyr` is a complex object, please add documentation for the parameter explaining what should be inside the object.